src/components/ErrorForm/ErrorFormInlineError.jsx

```diff
@@ -1,77 +1,59 @@
-import React from 'react'
+import React, { useRef, useState } from 'react'
 import formStyle from './errorForm.module.css'
 
-class ErrorFormInlineError extends React.Component {
-  emailRef = React.createRef()
+export default function ErrorFormInlineError() {
+  const emailRef = useRef(null)
+  const [emailInputValid, setEmailInputValid] = useState(false)
 
-  constructor(props) {
-    super(props)
-
-    this.state = {
-      emailInputValid: undefined,
-    }
-
-    this.validateEmail = this.validateEmail.bind(this)
-    this.handleChange = this.handleChange.bind(this)
-    this.handleBlur = this.handleBlur.bind(this)
-    this.handleSubmit = this.handleSubmit.bind(this)
-  }
-
-  validateEmail(value) {
+  function validateEmail(value) {
     return value !== ''
   }
 
-  handleChange() {
-    this.setState({ emailInputValid: true })
+  const handleChange = () => {
+    setEmailInputValid(true)
   }
 
-  handleBlur() {
-    const value = this.emailRef.current.value
-    const emailInputValid = this.validateEmail(value)
-
-    this.setState({ emailInputValid })
+  const handleBlur = () => {
+    const value = emailRef.current.value
```

Comment:
might be nice to destructure this 
```suggestion
    const { value } = emailRef.current
```